src/acceptance-test/java/de/zalando/aruha/nakadi/repository/db/EventTypeDbRepositoryTest.java

```diff
@@ -44,11 +44,7 @@ public void setUp() {
             repository = new EventTypeDbRepository(template, mapper);
             connection = datasource.getConnection();
 
-            ResourceDatabasePopulator rdp = new ResourceDatabasePopulator();
-            rdp.addScript(new ClassPathResource("schema.sql"));
-            rdp.populate(connection);
-
-            template.execute("TRUNCATE TABLE zsr_data.event_type");
+            template.execute("DELETE FROM zn_data.event_type");
```

Comment:
I'm gonna fix this once we merge this PR into the base branch. I need to add `DELETE` to other tests which were affecting this one. We can leave it here by now.